sources.cfg

```diff
@@ -1,13 +1,4 @@
 [buildout]
-extensions +=
-    mr.developer
-
-sources = sources
-
-[remotes]
-4tw_github = https://github.com/4teamwork
-4tw_github_push = git@github.com:4teamwork
-
-[sources]
-ftw.publisher.core = git ${remotes:4tw_github}/ftw.publisher.core.git pushurl=${remotes:4tw_github_push}/ftw.publisher.core.git
-ftw.publisher.sender = git ${remotes:4tw_github}/ftw.publisher.sender.git pushurl=${remotes:4tw_github_push}/ftw.publisher.sender.git
+extends =
+    http://kgs.4teamwork.ch/sources.cfg
+    https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/4teamwork-psc.cfg
```

Comment:
This is not necessary because `ftw.publisher.mailreport` should only have public dependencies. ✂️